src/com/mitrais/view/WelcomeScreen.java

```diff
@@ -0,0 +1,49 @@
+package src.com.mitrais.view;
+
+import src.com.mitrais.model.Account;
+import src.com.mitrais.utils.Helper;
+import src.com.mitrais.utils.Validation;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class WelcomeScreen extends BaseScreen {
+    private Scanner in = new Scanner(System.in);
+    private TransactionScreen transactionScreen = new TransactionScreen();
+    Validation validation = new Validation();
+
+    public void show(){
+        System.out.println("======================= ATM Simulation =======================");
+        System.out.println("Enter Account Number: ");
+
+        String accountNumber = in.nextLine();
+
+        List<String> errAccount = validation.ValidateAccountNumberOrPin(accountNumber, true);
+        if(!errAccount.isEmpty()){
+            for (String error : errAccount) {
```

Comment:
What is this cycle for? Why the need of using a for if you exit immediately on the first one?

Reply:
The intention of using two for loop is to validate and show error messages independently.
This behaviour intended to accommodate TC 01 to TC 05. Please correct me if I am wrong about that.

Reply:
You're not wrong about the requirement, but wrong about the implementation. It doesn't really make sense to have a for loop just to exit on the first iteration, right? 
You have to refactor that somehow, because the code is not correct